### ISSUE TITLE
Unused prometheus metrics

### ISSUE THREAD

**Issue by jayvdb** [geofmureithi/apalis#347]
Building emits the following warning

```
warning: unused `Counter` that must be used
  --> src/layers/prometheus/mod.rs:93:9
   |
93 |         metrics::counter!("requests_total", &labels);
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: counters do nothing unless you use them
   = note: `#[warn(unused_must_use)]` on by default
   = note: this warning originates in the macro `$crate::counter` which comes from the expansion of the macro `metrics::counter` (in Nightly builds, run with -Z macro-backtrace for more info)

warning: unused `Histogram` that must be used
  --> src/layers/prometheus/mod.rs:94:9
   |
94 |         metrics::histogram!("request_duration_seconds", &labels);
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: histograms do nothing unless you use them
   = note: this warning originates in the macro `$crate::histogram` which comes from the expansion of the macro `metrics::histogram` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by geofmureithi** [comment]
This is fixed in v0.6, I wouldnt mind a PR to support v0.5

**Comment by jayvdb** [comment]
gotcha.  I can see the fix in https://github.com/geofmureithi/apalis/compare/master...chore/v0.6.0
